CHANGELOG.rst

```diff
@@ -32,6 +32,9 @@ New Features
   adding and removing cookies in bulk, without triggering a header rewrite on each item.
   ``PHAB_ID=D318013``
 
+* finagle-core: Allow to not bypass SOCKS proxy for localhost by usingg the GlobalFlag
+  `-com.twitter.finagle.sockssocksProxyForLocalhost`
```

Comment:
typo here with an extra socks. (though in real life, you do want pairs of socks). same typo in the commit message.

also, mind sorting this entry up with the other finagle-core changes (line 22ish)